cmd/jsonnetdeps/cmd.go

```diff
@@ -0,0 +1,252 @@
+/*
+Copyright 2020 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/fatih/color"
+
+	"github.com/google/go-jsonnet"
+	"github.com/google/go-jsonnet/cmd/internal/cmd"
+)
+
+func version(o io.Writer) {
+	fmt.Fprintf(o, "Jsonnet static dependency parser %s\n", jsonnet.Version())
+}
+
+func usage(o io.Writer) {
+	version(o)
+	fmt.Fprintln(o)
+	fmt.Fprintln(o, "jsonnetdeps {<option>} <filename>")
```

Comment:
We don't have an established convention for naming the tools yet. What do you think about adding a dash, like `jsonnet-deps`, `jsonnet-lint`? It seems more readable to me than `jsonnetdeps`, `jsonnetlint`. The commands in the second style kinda blend together for me, especially with variable width font: jsonnetdeps, jsonnetlint.

I'm not sure. What do you think?

(Either way we're going to leave `jsonnetfmt` alone for now).

Reply:
Absolutely. `jsonnet-deps` was my first choice as well. I changed my mind after looking at `jsonnetfmt`.